tools/cypress.puml

```diff
@@ -0,0 +1,75 @@
+@startuml
+sprite $cypress [64x64/16] {
+0000000000000000000000000000000000000000000000000000000000000000
```

Comment:
This one looks good. Only the border looks weird but I think no one is going to argue.